dev/com.ibm.ws.microprofile.reactive.messaging_fat/fat/src/com/ibm/ws/microprofile/reactive/messaging/fat/apps/simple/SimpleRxMessagingServlet.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2019 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.reactive.messaging.fat.apps.simple;
+
+import static org.junit.Assert.assertEquals;
+
+import javax.inject.Inject;
+import javax.servlet.annotation.WebServlet;
+
+import org.junit.Test;
+
+import componenttest.app.FATServlet;
+
+@WebServlet("/")
+public class SimpleRxMessagingServlet extends FATServlet {
+
+    @Inject
+    private SimpleRxMessagingBean rxBean;
+
+    @Test
+    public void prod() {
+        assertEquals("PRODDED!", rxBean.prod());
```

Comment:
This needs to be a better name if the FAT bucket is not experimental.

Reply:
done ... rewrote the test a bit